lib/resque/unique_by_arity/modulizer.rb

```diff
@@ -44,6 +44,12 @@ def self.to_mod(configuration)
               Resque::UniqueByArity.debug("#{self}.unique_in_queue_redis_key for #{args_for_uniqueness} is: #{ColorizedString[key].green}")
               key
             end
+
+            define_method(:unique_in_queue_redis_value) do |queue, payload|
+              unique_hash, args_for_uniqueness = redis_unique_hash(payload)
+              Resque::UniqueByArity.debug("#{self}.unique_in_queue_redis_value for #{args_for_uniqueness} is: #{ColorizedString[unique_hash].green}")
+              unique_hash
+            end
```

Comment:
Just curious, you are overriding it here? where this method is defined?

Reply:
It's defined [here](https://github.com/teamintricately/resque-unique_in_queue/blob/master/lib/resque/plugins/unique_in_queue.rb#L27).

Unfortunately, these gems' architectures are not the best.